recovery/cmd/launchBackup.go

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright 2021 Red Hat, Inc.
```

Comment:
2022?

Reply:
Code was started last year in a dev repo. I meant to check for dates... I'll fix these up now